miniapp/src/test/java/com/rakuten/tech/mobile/miniapp/MiniAppDownloaderSpec.kt

```diff
@@ -78,10 +74,71 @@ class MiniAppDownloaderSpec {
         assertFalse { getMockedDownloader().isManifestValid(manifestEntity) }
     }
 
+    @Test
+    fun `when no existing app in local storage, run download execution`() {
+        runBlockingTest {
+            val apiClient: ApiClient = mock()
```

Comment:
You could setup these mocks at the top of this class and then you don't have to define them in every test.